docs/2.9.3/docs/tools/daml-shell/index.rst

```diff
@@ -50,11 +50,11 @@ Running Daml Shell
 ==================
 
 You can run Daml Shell from a ``jar`` artifact or from a ``Docker``
-image.  See the Release Notes for download instructions.
+image. See the Release Notes for download instructions.
```

Comment:
Like Carrie, I'm not sure what release notes refers to here, a link would be useful - I don't see any download instructions on the Github releases if that's where it's supposed to link to?